cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -229,18 +233,24 @@ func (s *Server) installedPackageDetail(ctx context.Context, name, namespace str
 	return &corev1.InstalledPackageDetail{
 		InstalledPackageRef: &corev1.InstalledPackageReference{
 			Context: &corev1.Context{
-				Namespace: namespace,
+				Namespace: name.Namespace,
 			},
-			Identifier: name,
+			Identifier: name.Name,
 			Plugin:     GetPluginDetail(),
 		},
-		Name:                  name,
+		Name:                  name.Name,
 		PkgVersionReference:   pkgVersion,
 		CurrentPkgVersion:     lastAppliedRevision,
 		ValuesApplied:         valuesApplied,
 		ReconciliationOptions: installedPackageReconciliationOptionsFromUnstructured(unstructuredRelease.Object),
 		AvailablePackageRef:   availablePackageRef,
 		// TODO (gfichtenholt) PostInstallationNotes
+		// looks like I may have to resort to getting the notes via helm APIs, flux doesn't do it
+		// see discussion in https://cloud-native.slack.com/archives/CLAJ40HV3/p1629244025187100
+		// bottom line: "helm get notes" is a read-only activity that you can replicate in your
+		// own code by copying the implementation, or calling to the library function if there is one
+		// https://github.com/helm/helm/blob/main/cmd/helm/get_notes.go.
```

Comment:
You can use the helm action in code the same way we do in the helm plugin, I would think. (Not necessary now, just related to your comment :) )

Reply:
yes, I plan on it